CV Backend/src/main/java/com/qa/demo/service/MongoUserDetailsService.java

```diff
@@ -0,0 +1,44 @@
+package com.qa.demo.service;
+
+import java.util.*;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.security.core.authority.SimpleGrantedAuthority;
+import org.springframework.security.core.userdetails.User;
+import org.springframework.security.core.userdetails.UserDetails;
+import org.springframework.security.core.userdetails.UserDetailsService;
+import org.springframework.security.core.userdetails.UsernameNotFoundException;
+import org.springframework.stereotype.Component;
+
+import com.qa.demo.domain.Account;
+import com.qa.demo.repository.AccountRepository;
+
+@Component
+public class MongoUserDetailsService implements UserDetailsService
```

Comment:
literal values
== null seen this throughout the code as well, make it into a helper method in a helper class

Reply:
done